src/components/AddPaymentMethodMenu.js

```diff
@@ -84,7 +84,11 @@ function AddPaymentMethodMenu({isVisible, onClose, anchorPosition, anchorAlignme
                           },
                       ]
                     : []),
-                ...(!ReportActionsUtils.hasRequestFromCurrentAccount(lodashGet(iouReport, 'reportID', 0), lodashGet(session, 'accountID', 0))
+
+                // Users can choose to pay with business bank account in case of Expense reports or in case of P2P IOU report
+                // which then starts a bottom up flow and creates a Collect workspace where the payer is an admin and payee is an employee.
+                ...(ReportUtils.isExpenseReport(iouReport) ||
+                (ReportUtils.isIOUReport(iouReport) && !ReportActionsUtils.hasRequestFromCurrentAccount(lodashGet(iouReport, 'reportID', 0), lodashGet(session, 'accountID', 0)))
```

Comment:
I think it would be much more clear to put this outside of the JSX - like make some variable(s) above the `return` and put the comment there... thoughts?